nestor_api/lib/config.py

```diff
@@ -46,6 +46,16 @@ def get_app_config(app_name: str, config_path: str = Configuration.get_config_pa
     return _resolve_variables_deep(config)
 
 
+def get_applications(app_config: dict) -> list:
```

Comment:
According to the docstring, I would rather name the function `get_application_processes`